consistency-tests/__tests__/ref-forwarded.test.tsx

```diff
@@ -6,6 +6,7 @@ import * as ReactDOM from "react-dom";
 import Breadcrumbs from "../../packages/wonder-blocks-breadcrumbs/src/components/breadcrumbs";
 import BreadcrumbsItem from "../../packages/wonder-blocks-breadcrumbs/src/components/breadcrumbs-item";
 import Button from "../../packages/wonder-blocks-button/src/components/button";
+import Checkbox from "../../packages/wonder-blocks-form/src/components/checkbox";
```

Comment:
forgot to mention this in my review but this path should be added to `tsconfig.json` in `consistency-tests/`. all the dependencies are supposed to be listed there. more context in this slack thread: https://khanacademy.slack.com/archives/C4PE1QM5Y/p1690396563362009